fastapi_sqla/aws_rds_iam_support.py

```diff
@@ -0,0 +1,30 @@
+import boto3
+from pydantic import BaseSettings
+from sqlalchemy import event
+
+
+def startup(engine):
```

Comment:
I do care 1/10 but it's not really starting anything
```suggestion
def configure(engine):
```

Reply:
I called it setup to be consistent with the lib.